docker/root/defaults/settings.json

```diff
@@ -3,6 +3,6 @@
   "baseURL": "",
   "address": "",
   "log": "stdout",
-  "database": "/database/filebrowser.db",
```

Comment:
This settings file is related to the linuxserver based container and the path of the file is set correctly

Reply:
It should probably be changed elsewhere then? The documentation seemed to vary between `/database/filebrowser.db` and `/database.db`? For example what caught me out was the Alpine example at https://filebrowser.org/installation#docker uses `/database.db` but then the text below to the sample config file at https://github.com/filebrowser/filebrowser/blob/master/docker/root/defaults/settings.json is then `/database/filebrowser.db`? Perhaps it just needs some text added to make it clear you need to change the sample config file to match? (I'll confess when I first set things up I had a mismatch and it took me a while to work out what was going on!)

Reply:
No, the default docker image https://github.com/filebrowser/filebrowser/blob/master/Dockerfile uses the https://github.com/filebrowser/filebrowser/blob/master/docker_config.json for config. The file you are referring to is used in the linuxserver based image. But I agree that the docs need to be changed. Feel free to make a [PR](https://github.com/filebrowser/docs/pulls)

Reply:
OK I'll try to look hopefully this weekend

Reply:
Sorry for the delay - got round to it in https://github.com/filebrowser/docs/pull/52 by tweaking the docs very slightly